lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -127,6 +127,25 @@ def test_masquerade(self):
         self.update_masquerade(username=user.username)
         assert self.client.get(self.url).data['dates_widget']['user_timezone'] == 'Asia/Tokyo'
 
+    @override_experiment_waffle_flag(COURSE_HOME_MICROFRONTEND, active=True)
+    @override_waffle_flag(COURSE_HOME_MICROFRONTEND_OUTLINE_TAB, active=True)
+    def test_course_staff_can_see_outline_in_masquerade(self):
+        """
+        Test that course staff can see the outline when masquerading as a learner
+        """
+        from common.djangoapps.student.roles import CourseInstructorRole
+        instructor = UserFactory(
+            username='instructor',
+            email=u'instructor@example.com',
+            password='foo',
+            is_staff=False
+        )
+        CourseInstructorRole(self.course.id).add_users(instructor)
```

Comment:
Interesting. Wasn't expecting the function `add_users` to allow a single object passed in. Would have thought you would need to wrap in a list or something.